backend/event-parse/parse.go

```diff
@@ -753,3 +753,57 @@ func UnmarshallMouseInteractionEvent(data json.RawMessage) (*MouseInteractionEve
 	}
 	return &aux, nil
 }
+
+type Event struct {
+	Type      EventType
+	Timestamp int64
+	Data      interface{}
+}
+
+func containsAnyOf(str string, substrings []string) bool {
+	for _, s := range substrings {
+		if len(s) > 0 && len(str) >= len(s) && str != s && (str == s || str[len(str)-len(s):] == s) {
```

Comment:
is this check different than `strings.Contains` ?

Reply:
It checks if a string contains any of the substrings in the array. I think I can substitute that inner if statement with strings.contains though.